### ISSUE TITLE
Enable choosing Directory in file uploader

### ISSUE THREAD

**Issue by joshuef** [maidsafe/safe_browser#291]
Using an input like so:
`<input type="file" webkitdirectory directory multiple>`

When choosing a directory, Peruse only shows the directory name in `FileList`:
![peruse_directory_input](https://user-images.githubusercontent.com/10050403/35813489-9740687e-0a48-11e8-80ce-c06ca509f86f.PNG)
![peruse_file_list](https://user-images.githubusercontent.com/10050403/35813506-a25903ce-0a48-11e8-97ca-ead62ebf71cd.PNG)

Whereas if I open the same running web app in Google Chrome, I see the expected output:
![google_chrome_directory_input](https://user-images.githubusercontent.com/10050403/35813557-c7af8364-0a48-11e8-802c-bdfcfba50fae.PNG)
![google_chrome_file_list](https://user-images.githubusercontent.com/10050403/35813561-caa1260e-0a48-11e8-8035-3ee0c1336c2c.PNG)

**Comment by joshuef** [comment]
This is an electron bug. Tracked here: https://github.com/electron/electron/issues/749

**Comment by joshuef** [comment]
As it's blocked. I'd opt to remove from this project.

@bochaco thoughts?

**Comment by bochaco** [comment]
yes, I guess we shouldn't include it in our project for now since its resolution date doesn't depend on us.

**Comment by hunterlester** [comment]
@bochaco @joshuef 
Right, not needed for the upcoming release.
This is an unfortunate issue because it prevents us from making web hosting manager into a web app.
I tried a work around using `dialog.showOpenDialog(mainWindow, { properties: ['openDirectory'] })` to no avail.

**Comment by hunterlester** [comment]
Providing further evidence to close this issue without implementation:
It's possible to implement statically in electron with a simple workaround: https://github.com/hunterlester/safe-app-base/compare/electron-poc
However, the workaround doesn't work for dynamic webview contents and even makes webviews buggy: https://github.com/maidsafe/safe_browser/compare/master...hunterlester:dir-example

It looks like Electron doesn't want to support this feature.
Proposal to close this issue.

**Comment by Sash1ok1** [comment]
Can you point to a file in electronjs, in which the file or directory is selected? I need to rebuild the electron in order for webkitdirectory to work in both Chrome and NW.js. Recursively get FileList, in name the name of the file, in webkitRelativePath the relative path to the file from the selected directory
